src/App.svelte

```diff
@@ -113,23 +98,18 @@
           addressOrName={$activeRouteStore.params.addressOrName}
           {wallet} />
       {:else if $activeRouteStore.resource === "404"}
-        <NotFound title="404" subtitle="Nothing here" />
+        <div class="wrapper" style="justify-content: center;">
```

Comment:
I think we should be able to either add another class here to avoid this inline CSS, or move the entire div into the component itself.
Or if there is no other way, maybe we can use sveltes style directives?

```suggestion
        <div class="wrapper" style:justify-content="center">
```

Reply:
b556f3b9587b96fa099dbadf7067cce88424c9f5